### DISCUSSION TITLE
simple:rest

### DISCUSSION THREAD

**Discussion by andreaciotti** [meteor/meteor#11679] [Compat Packages]
Hi,
a very used (and useful) package that does not work with Meteor 2.3+, and is abandoned by the developer, is simple:rest (https://github.com/stubailo/meteor-rest).
I think it could become a new Compat Package.

**Comment by StorytellerCZ** [comment]
There is my PR, which makes it work: https://github.com/stubailo/meteor-rest/pull/164
Since I don't see it being in consideration for MCP, I agree with this motion.

**Comment by andreaciotti** [comment]
I just see it was forked by Meteor Compat: https://github.com/meteor-compat/meteor-rest. Maybe you can create the PR here.

**Comment by filipenevola** [comment]
Hi @andreaciotti, please provide feedback.

See my [post](https://forums.meteor.com/t/feedback-about-simple-rest-compat/56804) in the Forums.

**Comment by andreaciotti** [comment]
Hi @filipenevola, I tried some publications and methods and they seem to work fine!

**Reply by mdoll-arcadis** [reply]
Thanks for this! I have an old app that uses simple:json-routes package that is part of this. Perhaps there is a better package for REST these days but it's nice to see this get updated.
